package.json

```diff
@@ -274,10 +274,12 @@
     "history": "^5.3.0",
     "node-fetch": "2.6.6",
     "query-string": "^7.1.1",
+    "randomatic": "^3.1.1",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-redux": "^7.2.6",
     "redux": "^4.1.2",
+    "socket.io": "^4.5.2",
```

Comment:
I think this is a merge problem, this shouldn't be a requirement for this change lol. And neither should randomatic (they're both collab server dependencies, not electron client dependencies)

Reply:
I've just reverted that package.json/yarn commit. 

I've merged develop into my branch again, but I'm still having these local test cases fail due to those dependencies. I know it's got nothing to do with my branch, just thought it might be an easy fix.

![image](https://user-images.githubusercontent.com/58348527/192128215-c5b84557-9c3a-49ce-ac9b-b07c98f008a3.png)

Reply:
`cd src/collabServer && yarn` should fix it